components/bpp-if/Kconfig

```diff
@@ -1,8 +1,8 @@
-menu "SHA2017 BPP"
+menu "BPP"
```

Comment:
Gaan we nog een betekenis bedenken voor "BPP"? / Gaat iemand deze optie ooit nog serieus gebruiken? Ik stem voor verwijderen.

Reply:
Mja . . ach . . het is een compile ding . . 

En . . eigenlijk zijn @ranzbak en ik stiekem met een AR spel bezig dat EVT er gebruik van gaat maken 😄